src/app/components/simulation/simulation-view.scss

```diff
@@ -18,75 +18,82 @@
     border-radius: 10px;
   }
 
-  .terrariumBackGround{
+  .ruler {
+    z-index: 2;
+    background-image: url("../../assets/terrarium/Ruler-Centimeter.svg");
+    width: 394px;
+    height: 469px;
     position: absolute;
-    top: 0px;
-    width:100%;
-    background-color:#D5C7E0;
-    height:100%;
-    z-index:0;
-    border-radius: 20px;
-
+    top: 0;
   }
-  .terrariumBackTable{
+
+  .terrariumBackGround{
     position: absolute;
-    bottom: 0px;
-    height:39%;
+    top: 0px;
+    height: 100%;
     width:100%;
-    background-color: #c48787;
-    z-index:0;
-    border-radius: 0px 0px 20px 20px;
-
+    background-image: url("../../assets/terrarium/Terrarium-Background.svg");
+    border-radius: 19px;
+    z-index: 1;
   }
 
   .terrarium{
     position: absolute;
+    top: 0;
     margin: 0 auto;
-    bottom: 15px;
-    width: 366px;
-    height: 238px;
 
     .terrariumBack{
-      background-image: url("../../assets/terrarium-back.png");
+      background-image: url("../../assets/terrarium/Terrarium-Back.svg");
       position: absolute;
-      width: 366px;
-      height: 238px;
+      top: 0;
+      width: 394px;
+      height: 496px;
       z-index: 1;
     }
-    //plant roots have z index 2
-    .terrariumWater{
-      background-image: url("../../assets/terrarium-water.png");
-      position: absolute;
-      right: 0px;
-      bottom: 0px;
-      width: 366px;
-      height: 238px;
-      z-index: 3;
-    }
-    .terrariumSoilDry{
-      background-image: url("../../assets/terrarium-soil-dry.png");
+
+    .soil {
       position: absolute;
-      right: 0px;
-      bottom: 0px;
-      width: 366px;
-      height: 238px;
-      z-index: 3;
+      top: 0;
+      width: 394px;
+      height: 469px;
+      z-index: 2;
+      &.full {
+        background-image: url("../../assets/terrarium/Soil-Water-High.svg");
+      }
+      &.some {
+        background-image: url("../../assets/terrarium/Soil-Water-Low.svg");
+
+      }
+      &.none {
+        background-image: url("../../assets/terrarium/Soil-Water-None.svg");
+      }
     }
-    .terrariumSoilWet{
-      background-image: url("../../assets/terrarium-soil-wet.png");
+
+    .light {
       position: absolute;
-      right: 0px;
-      bottom: 0px;
-      width: 366px;
-      height: 238px;
-      z-index: 3;
+      top: 0;
+      width: 394px;
+      height: 469px;
+      z-index: 2;
+      &.full {
+        background-image: url("../../assets/terrarium/Light-High.svg");
+      }
+      &.some {
+        background-image: url("../../assets/terrarium/Light-Low.svg");
+
+      }
+      &.none {
+        background-image: url("../../assets/terrarium/Light-none.svg");
```

Comment:
Looks like a case sensitivity issue here (should be `Light-None.svg`).  That is why I like using all lower case for file and folder names.  Macs are case insensitive so you don't get build problems but Linux is case sensitive so then the build breaks.